### ISSUE TITLE
Another contiguous pointer...

### ISSUE THREAD

**Issue by Jm56Z** [ElmerCSC/elmerfem#149]
I'm sincerely sorry to be such an annoying person, but gcc-fortran is really tough.
I get the same error as https://github.com/ElmerCSC/elmerfem/issues/148 in
```
elmerfem/fem/src/SParIterSolver.F90:2255:17:

     CM % Cols => SCols
```

...and there might be more as more code compiles...

**Comment by raback** [comment]
You're right. There could be more. This could take long. Perhaps you could try to add the CONTIG statements yourself and report all at once. My gcc version 5.4.0 is running happily.

**Comment by raback** [comment]
PS. This was just fixed at git.

**Comment by Jm56Z** [comment]
My computer must connect to a future internet, I have gcc 8.1.1 lmao

Just a question (I'm quite a newbie developer) : where do I add the CONTIG statements ??

**Comment by kitpaddel** [comment]
Relevant forum post: [http://www.elmerfem.org/forum/viewtopic.php?f=2&t=5218](http://www.elmerfem.org/forum/viewtopic.php?f=2&t=5218)

**Comment by juhanikataja** [comment]
PR #152 should fix this at least on gcc 8.1.0. Can you confirm this?
